lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -8615,6 +8615,10 @@ namespace Js
 
     Var InterpreterStackFrame::OP_Ld_A(Var aValue)
     {
+        if (VarIs<JavascriptBigInt>(aValue))
+        {
+            return JavascriptBigInt::CloneToScriptContext(aValue, this->GetScriptContext());
```

Comment:
Similar to another comment I made elsewhere, this makes me kind of uncomfortable. Here's an operation that was working consistently for every type (even Proxy!) and now it needs special handling for BigInt? I feel like this is a workaround for a more fundamental issue somewhere else.

Reply:
you are right. I will defer it to a future PR when I find a better approach